app/routers/credits.py

```diff
@@ -0,0 +1,31 @@
+from fastapi import APIRouter, Request
+import json
+from typing import List
+
+from loguru import logger
+from starlette.templating import _TemplateResponse
+
+from app.dependencies import templates
+
+router = APIRouter()
+
+
+def credits_from_json() -> List:
+    path = 'app/resources/credits.json'
```

Comment:
Can you please define `RESOURCES` variable in the `config.py.example` and to use it instead?

I believe this line can do:

`RESOURCES_DIR = pathlib.Path(__file__).parent / 'resources'`

Reply:
fixed :)